src/main/java/edu/harvard/iq/dataverse/timer/DataverseTimerServiceBean.java

```diff
@@ -120,13 +120,13 @@ public void handleTimeout(jakarta.ejb.Timer timer) {
         }
         
         try {
-            logger.log(Level.INFO,"Handling timeout on " + InetAddress.getLocalHost().getCanonicalHostName());
+            logger.log(Level.FINE,"Handling timeout on " + InetAddress.getLocalHost().getCanonicalHostName());
         } catch (UnknownHostException ex) {
             Logger.getLogger(DataverseTimerServiceBean.class.getName()).log(Level.SEVERE, null, ex);
         }
         
         if (timer.getInfo() instanceof MotherTimerInfo) {
-            logger.info("Behold! I am the Master Timer, king of all timers! I'm here to create all the lesser timers!");
+            logger.fine("Behold! I am the Master Timer, king of all timers! I'm here to create all the lesser timers!");
```

Comment:
I'm going to miss these :-(

Reply:
You can still have it - at a low price of one FINE logging setting.

Reply:
Ha! It's like an old friend at this point. 😄